python/lsst/the/monster/assemble.py

```diff
@@ -37,44 +39,75 @@ class AssembleMonsterRefcat:
     photometry from the DES-calibrated catalogs. The catalogs are read
     in reverse priority order so that the final fluxes for any object
     come from the top priority catalog. At the end the intermediate
-    calibrated catalogs are converted to the final fluxes (e.g. LSST).
+    calibrated catalogs are converted to the final fluxes in the target
+    systems specified in target_catalog_info_class_list.
 
     Parameters
     ----------
     gaia_reference_class : `RefcatInfo`
         The input Gaia DR3 RefcatInfo object.
     catalog_info_class_list : `list` [`RefcatInfo`]
         Reverse-priority list of catalog info classes for assembly.
+    target_catalog_info_class_list : `list` [`RefcatInfo`]
+        List of catalog info classes that will be output in final monster.
     monster_path_inp : `str`, optional
         Output monster path, overriding the class config.
+    do_u_band_slr : `bool`, optional
+        Perform u-band SLR using SDSS u and DES g & r bands
     testing_mode : `bool`, optional
         Enter testing mode for read_stars?
-    synth_system : `str`, optional
-        Synthetic system to do final conversion.
     """
     def __init__(self,
                  gaia_reference_class=GaiaDR3Info,
                  catalog_info_class_list=[VSTInfo, SkyMapperInfo,
-                                          PS1Info, GaiaXPInfo, DESInfo],
+                                          PS1Info, GaiaXPInfo, GaiaXPuInfo, DESInfo],
+                 target_catalog_info_class_list=[SynthLSSTInfo, LATISSInfo, DESInfo, SDSSuInfo],
```

Comment:
I'll need to go down to see how `SDSSuInfo` is used...

Reply:
Currently, the SDSSuInfo is used so that out ouput monster catalog has SDSS_u_flux/err/flag columns. 
But, do we also want to include sdss u measurements where we have them (and do they have higher priority than gaiaXPu?